### DISCUSSION TITLE
Update Docker Container

### DISCUSSION THREAD

**Discussion by Z3roX** [thijsvanloef/palworld-server-docker#305] [Q&A] [answered]
Hi,
i am running a palworld server with this docker container since a week or so (0.17.0). Now you ve added some cool new features like auto backup and auto restart, that i would like to use. how to correctly upgade to 0.22.0 with my current save games?

**Comment by thijsvanloef** [comment, marked as answer]
Hi @Z3roX, best thing to do, is create a backup of all your files and then running the following commands (if you have mounted your volume on your local filesystem):

- `docker compose down --rmi all`
- change image to `image: thijsvanloef/palworld-server-docker:latest` in `docker-compose.yml`
- `docker compose up -d`

Please make sure you are using an up to date version of the docker-compose.yml example https://github.com/thijsvanloef/palworld-server-docker/blob/main/docker-compose.yml
